src/containers/DeliveryDetails/index.jsx

```diff
@@ -9,13 +8,25 @@ import DayPickerWrapper from 'src/components/DayPickerWrapper';
 import Actions from 'src/reflux/Actions';
 import DeliveryStore from 'src/reflux/DeliveryStore';
 import T from 'src/utils/i18n';
+import Animate from 'src/utils/animate';
+import firebase from "firebase/app/index";
 
 class DeliveryDetails extends Reflux.Component {
   constructor(props) {
     super(props);
 
     this.store = DeliveryStore;
     this.storeKeys = ['deliveryAdditional', 'setDate', 'setTime'];
+    this.animation = new Animate();
+  }
+
+  componentDidMount() {
+    this.unregisterAuthObserver = firebase.auth().onAuthStateChanged((user) => Actions.onAuthStateChanged(user));
+    this.animation.animateInViewTransition();
```

Comment:
move this line back to the constructor please -> this.unregisterAuthObserver = firebase.auth().onAuthStateChanged((user) => Actions.onAuthStateChanged(user));

Its there due to a previously fixed issue, #36